parm/wflow/plot.yaml

```diff
@@ -34,8 +6,39 @@ metatask_plot_allvars:
       fhr: '{% for h in range(0, workflow.LONG_FCST_LEN+1) %}{{ " %03d" % h }}{% endfor %}'
       cycledef: '{% for h in range(0, workflow.LONG_FCST_LEN+1) %}{% if h <= workflow.FCST_LEN_CYCL|min %}forecast {% else %}long_forecast {% endif %}{% endfor %}'
     task_plot_allvars_mem#mem#_f#fhr#:
-      <<: *default_task
       command: '&LOAD_MODULES_RUN_TASK; "plot_allvars" "&JOBSdir;/JREGIONAL_PLOT_ALLVARS"'
+      account: '&ACCOUNT;'
+      attrs:
+        cycledefs: forecast
+        maxtries: '1'
+      envars:
+        GLOBAL_VAR_DEFNS_FP: '&GLOBAL_VAR_DEFNS_FP;'
+        USHdir: '&USHdir;'
+        PDY:
+          cyclestr:
+            value:  "@Y@m@d"
+        cyc:
+          cyclestr:
+            value:  "@H"
+        subcyc:
+          cyclestr:
+            value:  "@M"
+        LOGDIR:
+          cyclestr:
+            value:  "&LOGDIR;"
+        SLASH_ENSMEM_SUBDIR: '&SLASH_ENSMEM_SUBDIR;'
+        ENSMEM_INDX: '#mem#'
+        nprocs: '{{ task_plot_allvars.execution.batchargs.nodes * task_plot_allvars.execution.batchargs.tasks_per_node }}'
+        fhr: '#fhr#'
+      native: '{{ platform.SCHED_NATIVE_CMD }}'
+      nodes: '{{ task_plot_allvars.execution.batchargs.nodes }}:ppn={{ task_plot_allvars.execution.batchargs.tasks_per_node }}'
+      exclusive: '{{ task_plot_allvars.execution.batchargs.exclusive }}'
```

Comment:
Why was `exclusive` added to the `plot.yaml` file?  I don't see this requirement for the `plot_allvars` tasks in the current develop.  While it's inclusion doesn't adversely affect experiments, on machines where `exclusive` isn't supported, it leaves warning messages.